src/Extension/ExtensionManager.php

```diff
@@ -138,10 +138,18 @@ public function disable($name)
         $enabled = $this->getEnabled();
 
         if (($k = array_search($name, $enabled)) !== false) {
-            unset($enabled[$k]);
-
             $extension = $this->getExtension($name);
 
+            if (in_array('locale', $extension->keywords)) {
```

Comment:
I'd say it's a better idea to use: `if (in_array('flarum-locale', $extension->extra)) {`